03_myfavorite/src/App.jsx

```diff
@@ -0,0 +1,11 @@
+import GamePage from './Components/GamePage';
+
+function App() {
+  return (
+    <main>
+      <GamePage />
+    </main>
```

Comment:
```suggestion
    <GamePage />
```

`main` 태그를 `GamePage` 의 최상위 태그로 정의해줘도 좋을 것 같아!
현재 최상위 태그가 `div` 로 되어있어서 `main`으로 갈아끼워줘도 효율적일 것 같으네?